src/Catalog.API/Model/PaginationRequest.cs

```diff
@@ -1,3 +1,13 @@
-﻿namespace eShop.Catalog.API.Model;
+﻿using System.ComponentModel;
 
-public record PaginationRequest(int PageSize = 10, int PageIndex = 0);
+namespace eShop.Catalog.API.Model;
+
+public record PaginationRequest(
+    [property: Description("Number of items to return in a single page of results")]
+    [property: DefaultValue(10)]
+    int PageSize = 10,
```

Comment:
Hmmm....we should probably support default parameter values instead of relying on the attribute as well. 🤔